src/MultiValueEvents.tsx

```diff
@@ -50,9 +60,22 @@ export class MultiValueEvents {
         this.refresh(values);
         this._fired++;
         const formService = await WorkItemFormService.getService();
-        formService.setFieldValue(this.fieldName, values.join(";"));
+
+        await formService.setFieldValue(this.fieldName, values.join(";"));
+
         return new Promise<void>((resolve) => {
             this._onRefreshed = resolve;
         });
     }
+    private async _checkFieldType(): Promise<string> {
+        const formService = await WorkItemFormService.getService();
+        const inv = await formService.getInvalidFields();
+        if (inv.length > 0 && inv.some((f) => f.referenceName === this.fieldName)) {
+            const field = await getClient().getField(this.fieldName);
```

Comment:
Could we use form service? that should have cache as I presume?

Reply:
The form service does not populate the isPicklist property 😟

Reply:
However this code limits this call to only happen when there is write by the extension to a field that consequently errors so 99% time we do not make this call.

Goal here is to only fire when control is misconfigured and give a more helpful error message.

Reply:
Make sense - maybe worth a comment :)